### ISSUE TITLE
Amplify build fails due to what seems to be a type error

### ISSUE THREAD

**Issue by awsperez** [aws-samples/serverless-pdf-chat#6]
When building the Amplify Frontend environment I get the following error:

```
# Starting phase: build
                                 # Executing command: npm run build
2023-10-06T16:41:46.277Z [INFO]: > frontend@0.0.0 build
                                 > tsc && vite build
2023-10-06T16:41:50.729Z [INFO]: src/components/DocumentDetail.tsx(26,21): error TS2345: Argument of type 'string' is not assignable to parameter of type 'number'.
2023-10-06T16:41:50.730Z [INFO]: src/components/DocumentDetail.tsx(26,40): error TS2339: Property 'toString' does not exist on type 'never'.
                                 src/components/DocumentUploader.tsx(92,32): error TS2345: Argument of type 'number | undefined' is not assignable to parameter of type 'number'.
                                 Type 'undefined' is not assignable to type 'number'.
                                 src/components/DocumentUploader.tsx(92,52): error TS2339: Property 'toString' does not exist on type 'never'.
2023-10-06T16:41:50.755Z [ERROR]: !!! Build failed
2023-10-06T16:41:50.755Z [ERROR]: !!! Non-Zero Exit Code detected
```

Any idea what it could be?

Thanks

**Comment by awsperez** [comment]
I think the error could be that `filesize`  is missing in the repo.

**Comment by pbv0** [comment]
Looking into this and will have a fix in the next days.

**Comment by pbv0** [comment]
Fixed in 19fd762.
